assets/js/translator.js

```diff
@@ -147,6 +147,9 @@ if(modeEnabled('translation')) {
             if($('div#translateText').is(':visible')) {
                 translateText();
             }
+            else if($('div#docTranslation').is(':visible')) {
+                translateDoc();
+            }
```

Comment:
Hm, maybe you're rebasing with your master instead of `origin/master`? I think these changes are separate so we don't want to merge them.